languages/PLI.groovy

```diff
@@ -98,6 +110,7 @@ def createPLIParms(String buildFile, LogicalFile logicalFile) {
 	def cics = props.getFileProperty('pli_compileCICSParms', buildFile) ?: ""
 	def sql = props.getFileProperty('pli_compileSQLParms', buildFile) ?: ""
 	def errPrefixOptions = props.getFileProperty('pli_compileErrorPrefixParms', buildFile) ?: ""
+	def compileDebugParms = props.getFileProperty('pli_compileDebugParms', buildFile)
```

Comment:
This build property does not exist in the sample as of today - 'samples/application-conf/PLI.properties' .